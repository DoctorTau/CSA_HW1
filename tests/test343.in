251
-4636 -6617 -2043 -6320 -3570 1093 6390 873 5273 738 -2690 3990 4188 609 3180 -7962 4043 -642 -6304 -5643 -4174 7175 9414 5314 -6170 -5962 1256 480 8126 7513 -3598 773 -2157 6971 -4732 -4380 6610 8570 1577 9789 -1054 -276 -7775 1919 7070 7345 -1945 -5255 -6786 -6792 4864 4382 -3706 2905 136 4897 3115 -1457 4578 -8905 7072 2316 7139 -3624 -5397 2619 -2348 4446 -8443 -7037 3044 -9460 3001 2128 6303 513 3197 6532 248 -23 633 -3275 9758 -857 4182 465 8643 -5045 4124 8753 5545 5872 -9016 -9075 -3572 -5456 -7447 712 2599 7442 -8240 -2572 5502 -5402 -82 -6567 -3089 7339 -3561 -8257 9343 5885 6747 -1505 -6026 2826 6653 -3837 -1367 8143 7579 3860 -2935 -4936 -8954 3795 4974 -5526 4011 -3335 -271 4074 567 6329 5402 163 4195 -1755 971 8378 -889 -3914 -6803 8077 2110 7632 4804 -8495 3355 5064 186 -8887 -8111 -4398 -9119 -3491 -8845 -1919 -8082 -6824 -5822 -4557 -5083 5630 2422 1872 5411 -7156 -712 -8036 -8019 9658 6185 1113 4449 9885 1656 -7721 -1843 -4303 7643 -178 -1472 -6158 9056 -3776 -3067 -5909 -1261 9771 -2392 -7655 -9526 2325 -1252 791 6266 9454 -2780 -6534 2130 9869 -2185 9591 6139 9514 7235 -7096 -3353 -1941 -3954 5722 1071 3225 -1469 -9437 -5509 -2032 -7768 -4374 -6920 -1390 -1598 -7255 -5451 -229 5345 606 -4772 -8090 -206 7350 -5938 -3738 420 -2604 4685 -4068 9084 -158 -4373 -5010 5952 -556 -9464 6468 -2147 -6552 -3276 -3753 -9028 