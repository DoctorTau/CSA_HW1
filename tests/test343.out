-4636 -6320 -3570 -2690 -7962 -642 -6304 -4174 -6170 -5962 -3598 -4732 -4380 -1054 -276 -6786 -6792 -3706 -3624 -2348 -9460 -9016 -3572 -5456 -8240 -2572 -5402 -82 -6026 -4936 -8954 -5526 -3914 -4398 -8082 -6824 -5822 -7156 -712 -8036 -178 -1472 -6158 -3776 -2392 -9526 -1252 -2780 -6534 -7096 -3954 -2032 -7768 -4374 -6920 -1390 -1598 -4772 -8090 -206 -5938 -3738 -2604 -4068 -158 -5010 -556 -9464 -6552 -3276 -9028 